cli/daemon/dash/dashapp/src/components/Nav.tsx

```diff
@@ -1,165 +1,310 @@
-import React, {FunctionComponent, useEffect, useState} from "react";
-import {Link, useParams, useLocation, matchPath} from "react-router-dom"
-import {useConn} from "~lib/ctx"
-import logo from "../logo.svg"
-import wordmark from "../wordmark.svg"
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { Link, matchPath, useLocation, useParams } from "react-router-dom";
+import { useConn } from "~lib/ctx";
+import logo from "../logo.svg";
+import wordmark from "../wordmark.svg";
 
-const menuItems: {href: string; name: string, external?: boolean}[] = [
-  {href: "", name: "Requests"},
-  {href: "/api", name: "API Docs"},
-  // Hiding this for now
-  // {href: "/diagram", name: "System Diagram"},
-  {href: "https://encore.dev/docs", name: "Encore Docs", external: true},
-]
+const menuItems: {
+  href: string;
+  name: string;
+  external?: boolean;
+  badge?: string;
+}[] = [
+  { href: "", name: "Requests" },
+  { href: "/api", name: "API Docs" },
+  { href: "/flow", name: "Flow", badge: "New!" },
```

Comment:
This file (`Nav.tsx`) is mostly unchanged, it's formatting that makes it look like a lot have changed. I have added the `badge` property to the `menuItems` list